ui/lib/core/addon/components/choose-pgp-key-form.hbs

```diff
@@ -19,19 +19,22 @@
           )
         }}
       </p>
-      <h4 class="has-text-weight-bold is-size-7 has-bottom-padding-m is-fullwidth">
-        {{concat "PGP Key " this.pgpKeyFile.filename}}
-      </h4>
-      <Hds::Copy::Snippet
+      <Hds::CodeBlock
         class="has-bottom-margin-s"
-        @textToCopy={{this.pgpKey}}
-        @color="secondary"
-        @onError={{(fn
-          (set-flash-message "Clipboard copy failed. Please make sure the browser Clipboard API is allowed." "danger")
```

Comment:
Does this kind of error still display when a user's browser does not allow the Clipboard API? I see we're passing errors through for when the generation of the key fails, but what about the copy button? 

And maybe this isn't a concern anymore, but thought I'd ask.